From ba6baab1013b98c8c1646f9739afa182378cc5cc Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Thu, 29 Oct 2020 19:25:52 +0700
Subject: [PATCH 339/442] drm: rcar-du: Fix help of DRM_RCAR_LVDS config

Since DRM_RCAR_LVDS config is available for not only internal encoder
of R8A7790 but also external encoder, this help is fixed.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 drivers/gpu/drm/rcar-du/Kconfig | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/Kconfig b/drivers/gpu/drm/rcar-du/Kconfig
index c44c5fc..4abfb50 100644
--- a/drivers/gpu/drm/rcar-du/Kconfig
+++ b/drivers/gpu/drm/rcar-du/Kconfig
@@ -23,8 +23,10 @@ config DRM_RCAR_LVDS
 	select DRM_PANEL
 	depends on ARCH_R8A7790 || ARCH_R8A7791 || COMPILE_TEST
 	help
-	  Enable support for the R-Car Display Unit embedded LVDS encoders
-	  (currently only on R8A7790 and R8A7791).
+	  Enable support LVDS encoders.
+
+	  If you want to access LVDS encoder, you select y.
+	  If you don't want to access LVDS encoder, you select n.
 
 config DRM_RCAR_VSP
        bool "R-Car DU VSP Compositor Support"
-- 
2.7.4

